js/packages/teams-ai/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@microsoft/teams-ai",
     "author": "Microsoft Corp.",
     "description": "SDK focused on building AI based applications for Microsoft Teams.",
-    "version": "1.0.1",
+    "version": "1.1.0",
```

Comment:
do we want to bump the version right before we release it?

Reply:
I have to do it in this PR because it fails the samples CI otherwise (due to sym linking)

Reply:
I did it in this commit because the build was failing https://github.com/microsoft/teams-ai/pull/1142/commits/a338c3dad7859dbec0ed1b70ec7130b34188af6b